### DISCUSSION TITLE
How to Add CSS Styles to Canvas Pieces Based on Conditions

### DISCUSSION THREAD

**Discussion by codeHarvest** [flbulgarelli/headbreaker#84] [Q&A]
I am working on a project where I need to target specific pieces of a canvas/puzzle element. 
I can successfully retrieve these pieces and apply color styles to them using metadata. 
However, I am facing challenges in applying other CSS styles, 
such as hiding and showing these pieces based on certain conditions and adding box shadows.

Could someone please guide me on how to:

Hide and show canvas pieces based on specific conditions.
Add CSS styles like box shadow to these canvas pieces.
Any code examples or pointers to relevant documentation would be greatly appreciated. 
Thank you!

**Comment by flbulgarelli** [comment]
¡Hi @codeHarvest!

Unfortunately, the board is puzzle display is based on an HTML `Canvas`, so you can't use CSS properties to control how it looks. You'll need to control those aspects using the native HTML `Canvas` API and/or the [Konva](https://konvajs.org/) API - the default implementation of `headbreaker` renders a canvas using that library.
